juniper_official/vpn/evpn/EvpnInstanceInfo.yml

```diff
@@ -0,0 +1,25 @@
+---
+EvpnInstanceInfoTable:
+   rpc: get-evpn-instance-information
```

Comment:
What is the scale.
Specify XSLT for scale.

Reply:
I have to analyse some of the use cases and check response.  Meanwhile approve the MR and i will open task for XSLT